python/lsst/ts/MTAOS/MtaosCsc.py

```diff
@@ -63,6 +67,10 @@ def __init__(self, config_dir=None, initial_simulation_mode=0):
                          initial_state=salobj.State.STANDBY,
                          initial_simulation_mode=int(initial_simulation_mode))
 
+        # Logger attribute comes from the upstream Controller class
+        self.log = self._addLogWithFileHandlerIfDebug(outputLogFile=verbose)
```

Comment:
I tried to keep the `__init__` as simple as possible and put some business logic into other functions. This is what I learned from the book of clean code. The reader only cares about what attributes this class has instead of the logic behind (how to construct each attribute if the related method is complex).

Reply:
I agree that if a computation is complex or non-obvious then it is wise to abstract it away. But adding methods that perform trivial tasks can make it harder for the user to see what is going on.

I recommend trying to write code to be as simple and clear as possible. That makes it easier to understand and maintain.

Furthermore, any code that you can remove is code that no longer needs to be tested, documented and maintained.

It is a delicate balance.